cypress/integration/learn/common-components/helpButton.js

```diff
@@ -0,0 +1,47 @@
+/* global cy */
+
+describe('Help Button', () => {
+  before(() => {
+    cy.visit('/');
+  });
+
+  it('should be visible', () => {
+    cy.visit(
+      '/learn/responsive-web-design/basic-html-and-html5/say-hello-to-html-elements'
+    );
+    cy.get('#get-help-dropdown').scrollIntoView().should('be.visible');
+  });
+
+  it('should toggle the dropdown menu', () => {
+    cy.visit(
```

Comment:
Does this need to be visited for each `it`? Or, are you just doing it as a way to unclick elements?

Reply:
I cleaned up the logic 🙂